packages/core/admin/admin/src/components/UnstableGuidedTour/Step.tsx

```diff
@@ -135,23 +158,28 @@ const createStepComponents = (tourName: ValidTourName): Step => ({
             )}
             <Flex gap={2}>
               {showSkip && (
-                <Button
-                  variant="tertiary"
-                  onClick={() => dispatch({ type: 'skip_tour', payload: tourName })}
-                >
+                <Button variant="tertiary" onClick={handleSkipAction}>
                   <FormattedMessage id="tours.skip" defaultMessage="Skip" />
                 </Button>
               )}
               {to ? (
                 <LinkButton
                   tag={NavLink}
                   to={to}
-                  onClick={() => dispatch({ type: 'next_step', payload: tourName })}
+                  onClick={() => {
+                    trackUsage(GUIDED_TOUR_TRACKING_EVENTS[tourName]?.create);
+                    dispatch({ type: 'next_step', payload: tourName });
+                  }}
```

Comment:
If you are creating a handler for skip maybe create one for this too? Or do them both inline? I think consistency one way or another would be nice